src/wp-includes/theme-templates.php

```diff
@@ -154,8 +148,15 @@ function the_block_template_skip_link() {
 			top: 5px;
 			width: auto;
 			z-index: 100000;
-		}
-	</style>
+		}';
+
+	$handle = 'skip-link-styles';
+
+	wp_register_style( $handle, false );
+	wp_add_inline_style( $handle, $skip_link_styles );
+	wp_enqueue_style( $handle );
```

Comment:
I have feedback a bit similar to the above here:
* This function is for printing scripts and styles, but now it doesn't.
* Furthermore, what I think makes it worse here is that it now enqueues the style but prints the script, which is inconsistent within a single function.

We could try to find another solution, such as introducing new functions for this as well, but in this case here I'd prefer if we for now removed the changes and opened a follow up ticket specific to this function. That way, we can figure out how to address it once we've completed the general groundwork for both avoiding printing style tags and script tags directly. We could also try to solve it now, but I think that may block the rest of the work and I don't think it has to be all done at once, as long as we log follow-up tickets not to forget about it.

Reply:
I am going to disagree with you on this one. This function is pretty new and is different from the others. The other functions have a use outside of block themes. This one does not. There is no reason to use this function outside of the context of block themes. This is a safe change.